tests/phpunit/tests/test-create-delete-languages.php

```diff
@@ -171,4 +171,43 @@ public function test_language_properties_in_transient() {
 		$languages = get_transient( 'pll_languages_list' );
 		$this->assertEqualSets( $properties, array_keys( reset( $languages ) ) );
 	}
+
+	/**
+	 * @doesNotPerformAssertions
+	 */
+	public function test_create_secondary_language_with_yoast() {
+		// first language
+		$args = array(
+			'name'       => 'English',
+			'slug'       => 'en',
+			'locale'     => 'en_US',
+			'rtl'        => 0,
+			'flag'       => 'us',
+			'term_group' => 2,
+		);
+		self::$model->add_language( $args );
```

Comment:
You can test it however I am not sure that it's mandatory to create a first language to reproduce the issue. This is the case in Polylang because `PLL_Filters_Links` is not instantiated before a language is created. However as you instantiate it manually before creating the other language, it should be sufficient.

Reply:
Indeed, it is not mandatory, but if I want to test the passage in this part: https://github.com/polylang/polylang/blob/fix-yoast-notices/include/model.php#L93-L98, `$term_languages` must not be empty.